rebar.config

```diff
@@ -16,7 +16,7 @@
     {ip_vs_conn, {git, "https://github.com/mesosphere/ip_vs_conn.git", {branch, "master"}}},
     {riak_dt, ".*", {git, "https://github.com/dcos/riak_dt.git", {branch, "make-faster"}}},
     {lashup, ".*", {git, "https://github.com/dcos/lashup.git", {branch, "master"}}},
-    {mesos_state, {git, "https://github.com/dcos/mesos_state.git", {branch, "master"}}},
+    {mesos_state, {git, "https://github.com/dcos/mesos_state.git", {branch, "urbanserj/tls"}}},
```

Comment:
Do you plan to have this `urbanserj/tls` as a long living branch or after the mesos_state change merge to master this will be updated?

Reply:
This PR is related to https://github.com/dcos/mesos_state/pull/37 which would merge `urbanserj/tls` to `mesos_state`'s master, so this will be updated after that PR is merged.

Reply:
@orsenthil the code is ready for review, but I need to write a short comment about all of these changes. I hope we will merge it into dcos today or tomorrow.